rust/cymbal/src/types/mod.rs

```diff
@@ -179,7 +179,8 @@ impl FingerprintedErrProps {
             .exception_list
             .iter()
             .filter_map(|e| e.stack.as_ref())
-            .flat_map(Stacktrace::get_frames);
+            .flat_map(Stacktrace::get_frames)
+            .filter(|e| e.in_app);
```

Comment:
I opted to remove the non in-app functions from the materialization. It would make it impossible to tell which function / source was the deepest otherwise and don't think anyone would be searching on non in-app frames anyways. Other option is to add yet another materialized column but that feels like cheating at this stage